File: api/Models/Calendar/CalendarSchedule.cs
using System.Collections.Generic;
using Scv.Api.Models.Lookup;

namespace Scv.Api.Models.Calendar
{
    public class CalendarSchedule
    {
        public CalendarDay Today { get; set; }
        public List<CalendarDay> Days { get; set; } = [];
        public List<FilterCode> Activities { get; set; } = [];
        public List<FilterCode> Presiders { get; set; } = [];
    }
}

File: web/src/types/CalendarSchedule.ts
import { Activity } from './Activity';
import { CalendarDay } from './CalendarDay';
import { Presider } from './Presider';

export interface CalendarSchedule {
  today: CalendarDay;
  days: CalendarDay[];
  schedule: CalendarDay[];
  activities: Activity[];
  presiders: Presider[];
}
